app/models/middleware_server.rb

```diff
@@ -42,11 +42,11 @@ def in_domain?
 
   # Returns whether a server is immutable or not.
   #
-  # By default, we define all servers as being immutable, so that power
-  # operations are not allowed, and it is the provider responsability to
+  # By default, we define all servers as being mutable, so that power
```

Comment:
The changed code in this method looks fine, but the comment here seems wrong.

It doesn't look like it's defaulting to mutable.  It looks like it's defaulting to whatever the properties are.